docs/requirements.txt

```diff
@@ -4,5 +4,7 @@ sphinx_rtd_theme
 funcparserlib==1.0.0a0
 sphinxcontrib-seqdiag
 numpydoc
-
--r ../requirements.txt
```

Comment:
This is removed as it's now called separately in the yaml file. This is to preserve ordering of the installation so we can get `xcffib` before `cairocffi`.